src/txn/txn_rollback_to_stable.c

```diff
@@ -975,15 +925,8 @@ __rollback_to_stable_btree(WT_SESSION_IMPL *session, wt_timestamp_t rollback_tim
     /* There is nothing to do on an empty tree. */
     if (btree->root.page == NULL)
         return (0);
-    /*
-     * Ensure the eviction server is out of the file - we don't want it messing with us. This step
-     * shouldn't be required, but it simplifies some of the reasoning about what state trees can be
-     * in.
-     */
-    WT_RET(__wt_evict_file_exclusive_on(session));
```

Comment:
Why don't you do this step any more? Is it just an optimization? If so I'd prefer to do it later and keep this change conservative.

Reply:
No, it is not an optimization. With the change of reading all the required leaf pages into the cache to abort increases the cache usage and rollback to stable is not evicting any of the pages explicitly, so enabling eviction on the tree where the RTS is performing takes care of evicting the pages that are loaded by RTS operation.

or If we don't want the eviction to be enabled on the tree where rollback to stable is performing, we need to evict the leaf pages locally once the rollback to stable is finished.

Reply:
Ah, I see. That makes sense, it's a bit sad that we need to make a change this significant now.